Products/CMFPlone/controlpanel/browser/maintenance.pt

```diff
@@ -105,6 +105,10 @@
                             tal:condition="form_name"
                             tal:content="form_name">Form name</legend>
 
+                    <p>
+                        <strong i18n:translate="text_zope_database_name">Database name:</strong> <span tal:replace="view/dbName" />
```

Comment:
Even though the dbName is the name you've choosen for your ZODB? The value comes from the view and is ``self.portal()._p_jar.db().database_name``